src/main/java/org/cascadebot/cascadebot/CascadeBot.java

```diff
@@ -220,7 +220,7 @@ private void init() {
         Thread.setDefaultUncaughtExceptionHandler(((t, e) -> LOGGER.error("Uncaught exception in thread " + t, MDCException.from(e))));
         Thread.currentThread()
                 .setUncaughtExceptionHandler(((t, e) -> LOGGER.error("Uncaught exception in thread " + t, MDCException.from(e))));
-
+        
```

Comment:
wat

Reply:
ignore that, me and binary were testing something